<launch>
  <node pkg="communication" type="mavlink_uart" name="mavlink_uart_node" output="screen" args="1">
 
	<!-- Read Mavlink_mission parameters -->
	<rosparam command="load" file="$(find communication)/launch/qgc_leader.yaml" />

  </node>
</launch>
